src-ui/src/app/components/document-detail/document-detail.component.ts

```diff
@@ -292,6 +292,24 @@ export class DocumentDetailComponent implements OnInit, OnDestroy, DirtyComponen
     return this.documentListViewService.hasNext(this.documentId)
   }
 
+  hasPrevious() {
+    return this.documentListViewService.hasPrevious(this.documentId)
+  }
+
+  nextDoc() {
+    this.documentListViewService.getNext(this.document.id).subscribe((nextDocId: number) => {
+      this.router.navigate(['documents', nextDocId])
+      this.titleInput?.focus()
+    })
+  }
+  
+  previousDoc () {
+    this.documentListViewService.getPrevious(this.document.id).subscribe((prevDocId: number) => {
+      this.router.navigate(['documents', prevDocId])
+      this.titleInput?.focus()
```

Comment:
```suggestion
```

Reply:
good idea! I just used my mouse when testing, but having it more keyboard centric is always nice.